tests/mongodb.spec.js

```diff
@@ -0,0 +1,48 @@
+const should = require('should');
+const mongoose = require('mongoose');
+const dailyWeathers = require("../models/weatherSchema");
+
+describe("MongoDB", function() {
+  afterEach(function() {
+    if (this.connection != 'undefined') {
+      dailyWeathers.collection.drop();
+    }
+  });
+
+  after(function() {
+    if (this.connection != 'undefined') {
+      this.connection.connection.db.dropDatabase(function() {
+        console.log("dropped.");
```

Comment:
I forgot to remove this log which looks unnecessary. Maybe we can omit callback function entirely.